src/components/grid/grid-container/grid-container.component.js

```diff
@@ -5,17 +5,43 @@ import GridItem from '../grid-item';
 
 const GridContainer = (props) => {
   const {
-    children
+    children,
+    m,
+    ml,
+    mr,
+    mt,
+    mb,
+    gridGap
   } = props;
   return (
-    <GridContainerStyle data-component='grid'>
+    <GridContainerStyle
+      data-component='grid'
+      m={ m }
```

Comment:
You could just `{ ...props }` here rather than pass down these margin props individually, then you wouldn't need to destructure them above either. 

Unless you've done this specifically so all the other styled-system padding props aren't available in this component?

Reply:
I was thinking about doing it like that, but in in the acceptance criteria is stated that we should allow to change margin and gridGap. Passing it like that would also allow the user to change paddings.

Reply:
Ok if we want to actively stop the user changing paddings then this is fine! I'd probably add `mx` and `my` as well, might as well make use of all the `styled-system` margin props we've got

Reply:
please keep in mind that we are also passing the `grid-gap`, so if we pass that as `{ ...props }, we'll allow every grid layout prop to be changed

Reply:
Good point yeah we probably don't want that